app/root.css

```diff
@@ -181,7 +181,6 @@ button.icon-link,
 a:visited.icon-link {
   font-size: 0.6rem;
   text-transform: uppercase;
-  white-space: nowrap;
```

Comment:
this is all black magic for me, but I assume it works

Reply:
it's half black magic to me too  🪄 

the use of `overflow-x: hidden` was triggering a change of default rule for `overflow-y: auto` causing `body` and `html` to be both scrollable on mobile screens 🙀 

...so I had to find a different alternative for overflow-x and I pinpointed this issue as causing the horizontal scrolling issue (a few px of overflow-x on <1000px window) by disabling 1 rule at a time for "a few minutes"

...and then I had to figure out another nowrap equivalent, so I changed the previous `flex: 0` (with implied flex-basis as "0%") to `flex: none` (with implied flex-basis as "auto" which defaults to the width of the child before wrapping - at least when some particular constellation of stars is most to our code)

🤞 hopefully no further chain reaction 🍀